### PR TITLE
Tick version to 3.4.0, rerender for py36 builds.

### PR DESCRIPTION
**Pull request by pmlandwehr**

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -25,14 +26,13 @@ test:
...
     - gridfs
     - pymongo
-  requires:  # [not win]
-    - mongodb  # [not win]
-    - setuptools  # [not win]
 
 about:
   home: https://github.com/mongodb/mongo-python-driver
   license: Apache 2.0
   summary: Python driver for MongoDB http://www.mongodb.org
```
Comment (ericdill) [inline recipe/meta.yaml]: While you're in here, can you add 

```yaml
license_file: LICENSE
```

**Comment by ericdill** [comment]
It might be a good idea to kill `recipe/run_test.sh`. AFAICT it's (1) bad practice to run the test suite in conda-forge and (2) even worse practice to only run it on *one* platform. (Both of those are my fault)

**Comment by pmlandwehr** [comment]
In addition to the requested changes, I also added some run requirements that seemed to be required by the extras for the different platforms - that should hopefully fix the builds, but I'd  welcome some double-checks on adding them.

**Comment by pmlandwehr** [comment]
Seems like the official build of `pykerberos` is missing some for `py36`, so might need to get a recipe into conda-forge.

**Comment by pmlandwehr** [comment]
Just added a build - https://github.com/conda-forge/staged-recipes/pull/2701

**Comment by jakirkham** [comment]
Any interest in this still, @pmlandwehr. If so, we would need to fix the merge conflicts.

**Comment by enlighter** [comment]
Why hasn't this been corrected and merged. Now pymongo has moved along to 3.6 and conda base channel only has 3.4. Is there something I can do to help the process along?

**Comment by ericdill** [comment]
> Why hasn't this been corrected and merged

Presumably because there are literally thousands of recipes needing the attention of the maintainers.

> Is there something I can do to help the process along?

1. push https://github.com/conda-forge/staged-recipes/pull/2701 through staged-recipes if pymongo still needs pykerberos
2. then submit a PR to pymongo-feedstock with an updated version and rerender the feedstock

**Pull request closed without merging**